docs/config/dev.toml

```diff
@@ -26,5 +26,5 @@ ethereum_adapter_relayer = 'https://goerli-relayer.adex.network'
 creators_whitelist = []
 minimal_deposit = "0"
 minimal_fee = "0"
-token_address_whitelist = []
 validators_whitelist = []
+token_address_whitelist = []
```

Comment:
please add tokens to the whitelist for the known used stablecoins we use: DAI, USDT, USDC.
Also if this is a Table could you please use the alternative styling of:
```
[[token_address_whitelist]]
address = ...
min_token_units_for_deposit = ...
...
[[token_address_whitelist]]
address = ...
min_token_units_for_deposit = ...
...

```